api/lib/DBClient.js

```diff
@@ -4,21 +4,32 @@ const user = process.env.CLOUDANT_USER;
 const pw = process.env.CLOUDANT_PW;
 
 function DBClient() {
-    this._client = Cloudant({account:user, password:pw});
+    this._client = Cloudant({ account: user, password: pw });
 }
 
-DBClient.prototype.get = function(db, from, size) {
+DBClient.prototype.get = function (options) {
+    // The Cloudant api does the verification of the 'from' and 'size' already.
+    if (typeof options === "object") {
+        if (options.db) {
+            if (typeof options.db !== 'string') {
+                let err = new Error(`Wrong type for parameter DB. ` +
+                    `Received ${typeof options.db}, expected a string`);
+                throw err;
+            }
+        }
+        this.options = Object.assign(this.__proto__.options, options);
```

Comment:
I think this comes from the discussion we had on having a "default database" for a given client, right? Well, sorry about that, but I was not very clear when I was talking about that :(

The intention was to have a private property (DBClient._db) that could receive a db when instantiating this client, so for example you could have something like this:

```js
[...]

let menus = new DBClient('menus');
let users = new DBClient('users');

let myUSer = users.getById('abc');
let menu = menus.getById('xyz');

myUser.menus.append(menu);

users.update(myUser);

[...]
```

And so on... Notice that I didn't have to specify the DBs for `getById()` and `update()`, they were set by default when I first instantiated each client.

In order to achieve that, the DBClient constructor (The `function DBClient()`) must receive an optional parameter db, and save it like this `this._db = db || null;`.

Reply:
I opened a PR #18 for this! :)